documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,169 @@
+reports, or through manual checks by maintainers. The Mini Support Rotation
+(MSR) runner makes these checks as part of their daily responsibilities.
+
+While the MSR runner monitors our communication channels closely, **any online
+maintainer who sees something that might be an incident should follow the
+incident recording process outlined here.** This does not mean that the
+reporting maintainer is inherently responsible for stabilizing the incident or
+investigating the root cause. It only means that they are responsible for making
+the initial report on the incident. Additionally, maintainers who stabilize
+services may not be the ones to perform the investigation after stabilization
+has been reached. Every incident has a [**Lead**](#how-do-we-determine-the-lead)
+who is the person responsible for providing updates on the incident and
+delegating tasks. The Lead is also responsible for hosting the retrospective
+after the incident is resolved.
+
+## How do we _classify_ incidents?
+
+Incidents are classified in two ways:
+
+- **Severity** - The impact of the incident
+- **Status** - The current state of the incident
+
+### Severity
+
+Incidents are classified using one of three levels of severity. It is important
+to note that regardless of severity, working to stabilize services (or whatever
+harm mitigation is relevant for the incident) always **takes priority over
+regular project work and MSR duties.** Once services are stabilized, any ongoing
+investigations, preventative measures, or documentation changes will be
+prioritized by the team alongside our other work.
+
+This severity scale is intentionally distinct from our GitHub priority scale.
+Any GitHub issues created to _stabilize_ a Severity 1 or Severity 2 incident are
+marked as "critical" in GitHub. Other issues are prioritized at your own
+discretion.
+
+- **Severity 1**: Critical issue affecting all/most users and all/most
+  functionalities of Openverse.
+- **Severity 2**: Major issue affecting a large number of users or critical
+  functionality of Openverse.
+- **Severity 3**: Minor issue affecting a limited number of users or
+  non-critical functionality of Openverse.
+
+### Status
+
+- **Stabilization pending** - The incident is actively disrupting service.
+- **Stabilized** - The incident is no longer directly disrupting service. Root
+  causes may still be unknown.
+- **Under investigation** - The root causes of the incident are being
+  researched. Stabilized but not yet resolved.
+- **Resolved** - The incident is stable, and long-term fixes to prevent future
+  occurrences or other mitigations have been identified.
+- **Reviewed** - A retrospective has been completed for the incident.
+
+As you might expect, incidents can be _destabilized_ if they reoccur, and the
+state should roll back to "Stabilization pending". Some incidents may not
+require further investigation once they are stabilized and can be immediately
+resolved.
```

Comment:
What's the rationale for _destabilising_ an incident rather than treating it as a wholly new incident (at least initially)? It feels like a big decision to make during an incident ("is this spike in average response times caused by the same thing as last week or is it a new thing altogether?").

Reply:
I think an issue would be destabilized if the exact same behavior that caused the initial incident was exhibited after a fix (which was presumed to address the issue) was deployed. It seems more probable to me that a fix might not address the ongoing incident than that we would have two, separate incidents occur at the same time.

That said, I think there's some temporality to destabilization. For instance, if we're more than 48 hours from resolving an incident, a new incident should probably be created. I'll add some language to this extent, let me know if you feel there's still ambiguity after and I'll make an issue to clarify that more.